sysutils/stern/Portfile

```diff
@@ -16,15 +16,24 @@ long_description    Stern allows you to tail multiple pods on Kubernetes and \
                     multiple containers within the pod. Each result is color \
                     coded for quicker debugging.
 
-homepage            https://github.com/wercker/stern
+homepage            https://github.com/stern/stern
 
-checksums           rmd160  655f16bb2c359778e396fae746df0ad89ed159d7 \
-                    sha256  e42e343052f4de3c050d2c856139841ead24afdf08d40e4e2cdda1fcac687c77 \
-                    size    24394
+checksums           rmd160  32298f4ee835c906883fa0e9d5df6b053617980f \
+                    sha256  d7d0349f37aab5158bf667317a2b843c2244eec4dc6e39053d0a6faf5312af25 \
+                    size    97275
 
-depends_build-append port:govendor
+# Reproduce the release configuration from the upstream .goreleaser.yml file
+set build_date      [exec date +%FT%T%z]
```

Comment:
This could/should be done with native Tcl code rather than forking a separate executable.

Reply:
I copied this line from another Portfile. I'll take a look at this.